ocdskingfisherprocess/cli/commands/new_transform_upgrade_1_0_to_1_1.py

```diff
@@ -0,0 +1,31 @@
+import ocdskingfisherprocess.database
+import ocdskingfisherprocess.cli.commands.base
+from ocdskingfisherprocess.transform.upgrade_1_0_to_1_1 import Upgrade10To11Transform
+
+
+class NewTransformUpgrade10To11CLICommand(ocdskingfisherprocess.cli.commands.base.CLICommand):
```

Comment:
This command is almost identical to the other command. I wonder whether the command should be `new-transform`, which takes a `transform_type` as an argument.

Reply:
Is it ok to come back to this? As said in last P.R., I do want to look at how it's triggered anyway - we have already discussed user needs around this being triggered by default, for instance.

Reply:
Sure, let's make sure we come back to it.